underfs/hdfs/src/main/java/alluxio/underfs/hdfs/NoopHdfsAclProvider.java

```diff
@@ -25,4 +25,10 @@
   public AccessControlList getAcl(FileSystem hdfs, String path) throws IOException {
     return null;
   }
+
+  @Override
+  public void setAcl(FileSystem hdfs, String path, AccessControlList acl) throws IOException {
+    throw new UnsupportedOperationException("setAcl not supported");
```

Comment:
Should we just ignore this, instead of throwing an exception?

Reply:
done